domains/certificates/Query.API/API/Program.cs

```diff
@@ -57,7 +57,7 @@
 
 builder.Services.AddHealthChecks();
 
-builder.Services.AddIntegrationEventBus();
```

Comment:
We have a shared library named "IntegrationEvents" with events we expected to receive on the Integration Event Bus. Given that we here delete the concept "Integration Event Bus" from the code, should we not also do something with event(s) under the library "IntegrationEvents"?